src/components/dashboard/CircularProgressbarComponent.js

```diff
@@ -1,21 +1,21 @@
-import React, { useState } from "react";
+import React from "react";
 import { CircularProgressbar, buildStyles } from "react-circular-progressbar";
-import { CircularProgressbarWithChildren } from "react-circular-progressbar";
 import "react-circular-progressbar/dist/styles.css";
 
-function Circular_progress_bar_3(props) {
+function CircularProgressBar(props) {
   const percentage = props.percentage;
+  const { fontSize, width, height} = props.style
```

Comment:
you can also do something like this

const { style: { fontSize, width, height }} = props